SportBot.py

```diff
@@ -0,0 +1,110 @@
+import mysql.connector
+from dotenv import load_dotenv
+import math
+import os
+from typing import List, Any
+
+class SportBot:
+    def __init__(self) -> None:
+        load_dotenv()
+        self.mydb = mysql.connector.connect(
+            host=os.getenv("DB_HOST"),
+            user=os.getenv("DB_USER"),
+            password=os.getenv("DB_PASSWORD"),
+            database=os.getenv("DB_DATABASE")
+        )
+
+        self.conn = self.mydb.cursor()
+        self.conn.execute("CREATE TABLE IF NOT EXISTS workouts (id BIGINT AUTO_INCREMENT PRIMARY KEY, user_id BIGINT, time TEXT, distance FLOAT)")
+        self.conn.execute(
+            "CREATE TABLE IF NOT EXISTS goals (id BIGINT AUTO_INCREMENT PRIMARY KEY,user_id BIGINT, distance FLOAT)")
+
+
+
+    def save_workout(self, user_id: str, time: str, distance: str) -> None:
+        self.conn.execute(
+            "INSERT into workouts (user_id, time, distance) values (%s, %s, %s)", (user_id, time, distance))
+        self.mydb.commit()
+
+    def history_workout(self, user_id: str) -> List[Any]:
+        self.conn.execute("SELECT * FROM workouts where user_id = %s", (user_id,))
+        workouts = self.conn.fetchall()
+        return workouts
+
+    def stats_workout(self, user_id: str) -> List[Any]:
+        self.conn.execute("SELECT * FROM workouts where user_id = %s", (user_id,))
+        workouts = self.conn.fetchall()
+        if len(workouts) == 0:
+            return
+        sum_ore = 0
+        sum_km = 0
+        for workout in workouts:
+            print("Tempo: ", workout[2], self.convert_time_in_sec(workout[2]))
```

Comment:
These print may be removed or replaced with a [logger](https://docs.python.org/3/library/logging.html)